hopsworks-web/yo/app/views/home.html

```diff
@@ -1,14 +1,14 @@
-<div class="container" ng-controller="MainCtrl as mainCtrl" style="width: 1500px;">
+<div ng-controller="MainCtrl as mainCtrl">
   <div ng-include="'nav.html'"></div>
   <div ng-if="mainCtrl.searchTerm.length > 3" class="row">
     <div id="projectSearchResults">
       <div ng-include="'views/search.html'"></div>
     </div>
   </div>
 
-  <div ng-if="mainCtrl.searchTerm.length <= 3" class="row" ng-controller="HomeCtrl as homeCtrl">
-    <div class="col-sm-6 col-md-2 col-lg-2" id="toursBox">
-      <div class="col-md-10 sidebar pull-left" style="min-width: 300px; max-width: 300px">
+  <div class="container content-wrapper" ng-if="mainCtrl.searchTerm.length <= 3" ng-controller="HomeCtrl as homeCtrl">
```

Comment:
Can we change the searchTerm length to '<=2'. Some people will search with 2 chars.

Reply:
This can be done but I do not know if it is appropriate to do it on this jira.

Reply:
ok. ask mahmoud about it - whether 2 chars is ok.